jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudwatch/logs/DownloadLogStream.kt

```diff
@@ -140,7 +141,7 @@ class LogStreamDownloadToFileTask(
         val descriptor = FileSaverDescriptor(message("cloudwatch.logs.download"), message("cloudwatch.logs.download.description"))
         val saveLocation = withContext(edt) {
             val destination = FileChooserFactory.getInstance().createSaveFileDialog(descriptor, project)
-            destination.save(null, logStream)
+            destination.save(null as VirtualFile?, logStream)
```

Comment:
Does this need a fix when min?

Reply:
Nah, they overloaded the method leading to us having to specify the type. I dont think we will ever remove it.

Reply:
Makes sense